optionals/variants_melb/CfgVehicles.hpp

```diff
@@ -8,6 +8,7 @@ class CfgVehicles {
 
     // Theseus Black
     class CLASS(MELB_I_AH6M_H_Theseus_Black): MELB_AH6M_H {
+        MACRO_SET_VEHICLE_CARGO
```

Comment:
What do MELBs have inside by default that doesn't fit a PMC?

Reply:
Some reason thought MELBs came with more stuff when I added the macro. Checked now, it comes with 4 first aid kits already, as well as a GPS and toolkit. I'll remove the macro.